client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -74,6 +76,16 @@ export function SitesDashboard( { queryParams: { search, status = 'all' } }: Sit
 
 	const selectedStatus = statuses.find( ( { name } ) => name === status ) || statuses[ 0 ];
 
+	const displayMode = useSitesDisplayMode();
+
+	const content =
+		// eslint-disable-next-line no-nested-ternary
+		displayMode === 'list' ? (
+			<SitesTable sites={ filteredSites } />
+		) : displayMode === 'tile' ? (
+			<SitesTileGrid sites={ filteredSites } />
+		) : null;
+
```

Comment:
Instead of using a nested ternary condition, could we set one mode by default?

I would prefer to set the right component.

```suggestion
const Sites = displayMode === 'tile' ? SitesTileGrid : SitesTable;
```

And then render the variable component in the return as we usually do. 
```diff
					) }
					{ filteredSites.length > 0 ? (
-						content
+						<Sites sites={ filteredSites } />
					) : (
```




**OPTION TWO** 

I prefer my first option, but If we cannot set a `displayMode` as default, or we expect to have more displayModes, then I suggest displaying them directly like this:

```patch
diff --git a/client/sites-dashboard/components/sites-dashboard.tsx b/client/sites-dashboard/components/sites-dashboard.tsx
index 4fa21e4c5f..2c4c96bb59 100644
--- a/client/sites-dashboard/components/sites-dashboard.tsx
+++ b/client/sites-dashboard/components/sites-dashboard.tsx
@@ -78,14 +78,6 @@ export function SitesDashboard( { queryParams: { search, status = 'all' } }: Sit
 
 	const displayMode = useSitesDisplayMode();
 
-	const content =
-		// eslint-disable-next-line no-nested-ternary
-		displayMode === 'list' ? (
-			<SitesTable sites={ filteredSites } />
-		) : displayMode === 'tile' ? (
-			<SitesTileGrid sites={ filteredSites } />
-		) : null;
-
 	return (
 		<main>
 			<PageHeader>
@@ -106,7 +98,10 @@ export function SitesDashboard( { queryParams: { search, status = 'all' } }: Sit
 						/>
 					) }
 					{ filteredSites.length > 0 ? (
-						content
+						<>
+							{ displayMode === 'list' && <SitesTable sites={ filteredSites } /> }
+							{ displayMode === 'tile' && <SitesTileGrid sites={ filteredSites } /> }
+						</>
 					) : (
 						<NoSitesMessage
 							status={ selectedStatus.name }


```